lte/gateway/python/magma/enodebd/enodeb_status.py

```diff
@@ -513,80 +553,82 @@ def _get_gps_status_as_bool(enodeb: EnodebAcsStateMachine) -> bool:
 ]:
     """
     Read the GPS coordinates of the enB from its configuration or the
     cached coordinate file if the preceding read fails. If reading from
     enB configuration succeeds, this method will cache the new coordinates.
 
     Returns:
-        (str, str): GPS latitude, GPS longitude
+        (str, str, str): GPS latitude, GPS longitude
     """
-    lat, lon = '', ''
+    lat, lon, ait = '', '', ''
     try:
         lat = enodeb.get_parameter(ParameterName.GPS_LAT)
         lon = enodeb.get_parameter(ParameterName.GPS_LONG)
+        ait = enodeb.get_parameter(ParameterName.GPS_AITI)
 
-        if lat != _gps_lat_cached or lon != _gps_lon_cached:
-            _cache_new_gps_coords(lat, lon)
-        return lat, lon
+        if lat != _gps_lat_cached or lon != _gps_lon_cached or lon != _gps_ait_cached:
+            _cache_new_gps_coords(lat, lon, ait)
+        return lat, lon, ait
     except (KeyError, ConfigurationError):
         return _get_cached_gps_coords()
     except ValueError:
-        logger.warning('GPS lat/long not understood (%s/%s)', lat, lon)
-        return '0', '0'
+        logger.warning('GPS lat/long/ait not understood (%s/%s/%s)', lat, lon, ait)
+        return '0', '0', '0'
 
 
-def _get_cached_gps_coords() -> Tuple[str, str]:
+def _get_cached_gps_coords() -> Tuple[str, str, str]:
     """
     Returns cached GPS coordinates if enB is disconnected or otherwise not
     reporting coordinates.
 
     Returns:
-        (str, str): (GPS lat, GPS lon)
+        (str, str, str): (GPS lat, GPS lon, GPS ait)
     """
     # pylint: disable=global-statement
-    global _gps_lat_cached, _gps_lon_cached
-    if _gps_lat_cached is None or _gps_lon_cached is None:
-        _gps_lat_cached, _gps_lon_cached = _read_gps_coords_from_file()
-    return _gps_lat_cached, _gps_lon_cached
+    global _gps_lat_cached, _gps_lon_cached, _gps_ait_cached
+    if _gps_lat_cached is None or _gps_lon_cached is None or _gps_ait_cached is None:
+        _gps_lat_cached, _gps_lon_cached, _gps_ait_cached = _read_gps_coords_from_file()
+    return _gps_lat_cached, _gps_lon_cached, _gps_ait_cached
 
 
 def _read_gps_coords_from_file():
     try:
         with open(CACHED_GPS_COORD_FILE_PATH, encoding="utf-8") as f:
             lines = f.readlines()
-            if len(lines) != 2:
+            if len(lines) != 3:
                 logger.warning(
-                    'Expected to find 2 lines in GPS '
+                    'Expected to find 3 lines in GPS '
                     'coordinate file but only found %d',
                     len(lines),
                 )
-                return '0', '0'
+                return '0', '0', '0'
             return tuple(map(lambda l: l.strip(), lines))
     except OSError:
         logger.warning('Could not open cached GPS coordinate file')
-        return '0', '0'
+        return '0', '0', '0'
 
 
-def _cache_new_gps_coords(gps_lat, gps_lon):
+def _cache_new_gps_coords(gps_lat, gps_lon, gps_ait):
```

Comment:
add type hint here 
```suggestion
def _cache_new_gps_coords(gps_lat: str, gps_lon: str, gps_ait: str):
```

Reply:
![image](https://user-images.githubusercontent.com/87301024/147812367-9946098e-6e88-41b2-b41a-aef744912c3e.png)
modified, thank you!